qt-downloader

```diff
@@ -202,7 +202,7 @@ def get_modules_info(url, version, toolchain):
     archives = package.xpath('DownloadableArchives/text()')[0].split(', ')
     if len(name_parts) == 4:
       info['main'] = (name, version, archives)
-    elif len(name_parts) == 5 and name_parts[3] != 'debug_info':
+    elif len(name_parts) == 5 and (with_debug_files or name_parts[3] != 'debug_info'):
```

Comment:
@herrfrei Should we swap the predicates? I.e. `name_parts[3] != 'debug_info' or with_debug_files`. Thus, `with_debug_files ` is respected only in the `name_parts[3] != 'debug_info'` case. Otherwise, as far as I understand, `info['addons']` will be populated disregarding the `name_parts[3]` value whenever `with_debug_files` is `True`.

Reply:
@dvolosnykh For me the order makes no difference. If debug_info is false, all other existing addons are added to info["addons"]. If it is true, the 'debug_info' addon is also added. This change only ensures that debug_info is added to the list. So yes, we can change it.

Reply:
I see. I am okay with the current state, then.